crates/bevy_macro_utils/src/official_bevy_crates.rs

```diff
@@ -0,0 +1,49 @@
+// This file is automatically generated by build.rs
+
+/// List of official bevy crates.
+/// This is used to correctly resolve re-exported third-party crates starting with `bevy_`.
+pub const OFFICIAL_BEVY_CRATES: &[&str] = &[
```

Comment:
@mockersf can we share this list between this and our publishing tooling somehow?